src/utilities/createRouteMethods.spec.ts

```diff
@@ -17,7 +17,7 @@ test.each([
   ] as const satisfies Routes
   const resolved = resolveRoutes(routes)
 
-  const response = createRouteMethods<typeof routes>(resolved)
+  const response = createRouteMethods<typeof routes>(resolved, vi.fn())
```

Comment:
`createRouteMethods` now requires `RouterPush` is passed as the second argument. So all these tests have to mock it. Felt gross updating these tests with an empty mock. But when I wrote the new tests at the bottom of this file it was really nice to be able to mock it.